scripts/hash-exec-copy.py

```diff
@@ -1,104 +1,197 @@
-#! /usr/bin/env python3
+#!/usr/bin/env python3
+"""
+Executive Vote Hash Generator
+
+This script fetches an executive vote document from the sky-ecosystem/executive-votes repository
+for a given date and calculates its keccak hash.
+
+Usage:
+    ./hash-exec-copy.py <date> OR
```

Comment:
I don't really understand the use of "copy" throughout this doc....  From the user perspective, nothing is being copied. I suggest maybe renaming the file and script to `hash-exec-sheet` or something along those lines.

Reply:
The term is "Executive Copy", it's not the action "copy", but a noun 😅